uwp/Hub-Tile/SfMosaicTile.md

```diff
@@ -108,7 +108,7 @@ Set the properties Header, ImageList as given below:
 
 {% endtabs %}
 
-![Hubtile displayed collection images](SfMosaicTile-images/SfMosaicTile-img1.jpeg)
+![Add image list in UWP SFMosaicTile](sfmosaictile-images/uwp-hub-tile-add-image-list-in-sfmosaictile.jpeg)
```

Comment:
remove uwp-hub-tile in image name. 
mosaictile is the control name,
add-image-list-in-uwp-sfmosaictile

Change the other image name as per the above.